sdk/cosmosdb/cosmos/review/cosmos-node.api.md

```diff
@@ -2116,6 +2118,8 @@ export interface RequestOptions extends SharedOptions {
     contentResponseOnWriteEnabled?: boolean;
     disableAutomaticIdGeneration?: boolean;
     enableScriptLogging?: boolean;
+    // (undocumented)
+    excludedLocations?: string[];
```

Comment:
we can consider it moving to sharedOptions as this might be applicable acrss all the requests

Reply:
Done